docs/sphinx/targets/python/pasqal.py

```diff
@@ -0,0 +1,65 @@
+import cudaq
+from cudaq.operator import *
+
+# This example illustrates how to use Pasqal's EMU_MPS emulator over Pasqal's cloud via CUDA-Q.
+
+# To obtain the authentication token we recommend logging in
+# by using Pasqal's Python SDK.
+# Fill in the  username and password via the environment variables.
+# The password can be left empty in an interactive session to be
+# prompted to enter the it securely via the command line interface.
+#
+# Contact Pasqal at help@pasqal.com or through https://community.pasqal.com for assistance.
+#
+# See our general docs https://docs.pasqal.com/cloud/set-up/
+# to see how to get this setup, or the CUDA-Q documentation at
+# https://nvidia.github.io/cuda-quantum/latest/using/backends/hardware/neutralatom.html#pasqal
+from pasqal_cloud import SDK
+import os
+
+sdk = SDK(
+    username=os.environ.get("PASQAL_USERNAME"),
+    password=os.environ.get("PASQAL_PASSWORD", None),
+)
+token = sdk._client.authenticator.token_provider.get_token()
+
+os.environ["PASQAL_AUTH_TOKEN"] = str(token)
+
+cudaq.set_target("pasqal",
+                 machine=os.environ.get("PASQAL_MACHINE_TARGET", "EMU_MPS"))
+
+# ```
+# cudaq.set_target("pasqal", machine="FRESNEL") ## To target QPU
```

Comment:
Is there an external link we can add to the docs that will always have the up to date machine names?

Reply:
This is available inside our cloud portal once logged in.
I could add sentence on this somewhere :)